app/kafka/manager/KafkaManager.scala

```diff
@@ -196,6 +196,25 @@ class KafkaManager(akkaConfig: Config) {
     tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName,KSGetTopics))(identity[TopicList])
   }
 
+  def getTopicListWithMoreInfo(clusterName: String) : Future[ApiError \/ TopicListWithMoreInfo] = {
+    val futureTopicIdentities = tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName,BVGetTopicIdentities))(identity[IndexedSeq[TopicIdentity]])
+    val futureTopicList = tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName,KSGetTopics))(identity[TopicList])
+
+    implicit val ec = apiExecutionContext
+    futureTopicIdentities.flatMap[ApiError \/ TopicListWithMoreInfo] { errOrTi =>
+      errOrTi.fold(
+      { err: ApiError =>
+        Future.successful(-\/[ApiError](err))
+      }, { ti: IndexedSeq[TopicIdentity] =>
+        val futureTopicList = tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName,KSGetTopics))(identity[TopicList])
+        futureTopicList.map{
+          case -\/(e) => -\/(e)
+          case \/-(tl) => \/-(TopicListWithMoreInfo(ti.filter(i => tl.list.contains(i.topic)), tl.deleteSet))
```

Comment:
Since the TopicList is the most current, we should use that as source for our list and add on the topic identity as an option.  This way, when we get the topic identity at a later time, it just gets used by the view if there, otherwise, we just show the topic name only.  So something like IndexedSeq[(String, Option[TopicIdentity])].